nodejs/make-data-key.js

```diff
@@ -1,18 +1,18 @@
-const { readMasterKey, ClientBuilder } = require("./helpers")
+const { readMasterKey, CsfleHelper } = require("./helpers")
 
 const localMasterKey = readMasterKey()
 
-const clientBuilder = new ClientBuilder({
+const csfleHelper = new CsfleHelper({
```

Comment:
again, I feel like all the "hard stuff" is happening in some other class. Is this file supposed to teach a user how to make a data key? Why not just move the contents of `findOrCreateDataKey` here?